src/ubuntu/16.04/Dockerfile

```diff
@@ -47,3 +45,17 @@ RUN apt-get install -y gettext \
             libunwind8 \
             uuid-dev \
     && apt-get clean
+
+# Install Azure CLI
```

Comment:
Any reason the Azure CLI is installed after the CoreCLR and CoreFX dependencies here whereas it is installed before in the other Ubuntu Dockerfiles?

Reply:
Due to how the commands are being executed in this Dockerfile, we'd have to execute another `apt-get update` at the beginning of the CoreCLR and CoreFX dependencies.

Reply:
Is this due to removing the Azure CLI feed registration? What is the reason for doing that?  It doesn't feel like a standard practice.  Aside from that, the existing pattern used here seems broken as each section should be doing an apt-get update if it is doing an install.